packages/clerk-js/src/ui/Components.tsx

```diff
@@ -245,13 +248,54 @@ const Components = (props: ComponentsProps) => {
       setState(s => ({ ...s, ...restProps, options: { ...s.options, ...restProps.options } }));
     };
 
-    componentsControls.closeModal = name => {
+    componentsControls.closeModal = (name, options = {}) => {
+      const { notify = true } = options;
       clearUrlStateParam();
-      setState(s => ({ ...s, [name + 'Modal']: null }));
+      setState(s => {
+        function handleCloseModalForExperimentalUserVerification() {
+          // @ts-ignore Expected value s['userVerificationModal']
+          const modal = s[name + 'Modal'] || {};
```

Comment:
Would ``s[`${name}Modal`]`` work for TS interpolation (and ts-ignore can be removed)?

Reply:
Worked, thanks !